OpenTabletDriver.Tools.udev/Program.cs

```diff
@@ -32,6 +32,16 @@ static void WriteRules(DirectoryInfo directory)
                 "https://github.com/OpenTabletDriver/OpenTabletDriver"
             );
 
+            //Prevent OpenTabletDriver from being recognized as joysticks.
+            Console.WriteLine(
+                new Rule(
+                    new Token("KERNEL", Operator.Equal, "js[0-9]*"),
+                    new Token("SUBSYSTEM", Operator.Equal, "input"),
+                    new ATTRS("name", Operator.Equal, "OpenTabletDriver Virtual Tablet"),
+                    new Token("RUN", Operator.Add, "/bin/rm %E{DEVNAME}")
```

Comment:
```suggestion
                    new Token("RUN", Operator.Add, "/usr/bin/env rm %E{DEVNAME}")
```

Probably this to maintain compatibility with distros like NixOS.